tm2/pkg/p2p/node_info.go

```diff
@@ -45,6 +45,9 @@ type NodeInfo struct {
 type NodeInfoOther struct {
 	TxIndex    string `json:"tx_index"`
 	RPCAddress string `json:"rpc_address"`
+	OS         string `json:"os"`
```

Comment:
Okay 👍 I did it that way because I wasn't sure that the node and the agent will always run on the same machine.
I'll make the change to fetch the information about the location and the system from the agent.